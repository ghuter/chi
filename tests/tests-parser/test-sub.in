x :: tab[1];
y :: tab[x];
z :: tab[a + b * c];
