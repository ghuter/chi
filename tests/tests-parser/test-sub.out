const SDECL(x, ESUBSCR(EMEM(tab) : ?[ECSTI(1)]) : ) : ?
const SDECL(y, ESUBSCR(EMEM(tab) : ?[EMEM(x) : ?]) : ) : ?
const SDECL(z, ESUBSCR(EMEM(tab) : ?[EBINOP(ADD, EMEM(a) : ?, EBINOP(MUL, EMEM(b) : ?, EMEM(c) : ?) : ?) : ?]) : ) : ?
